test/integration/appdeployment/application_suite_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package application_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+	"github.com/kubeedge/kubeedge/test/integration/utils/edge"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//context to load config and access across the package
+var (
+	ctx *edge.TestContext
+	cfg edge.Config
+)
+
+//Function to handle app deployment/delete deployment.
+func HandleDeployAndDeleteApplication(operation string, UID string, ImageUrl string) bool {
```

Comment:
better we move these methods to util / helpers . the test code can simply use the utility methods to perform such operations. also we can rename to HandleAddAndDeletePods  ? Since this method creates and deletes pod objects.